src/controllers/source.ts

```diff
@@ -28,6 +28,11 @@ export class SourceController {
       );
       ctx.body = resplist;
     } catch (err: any) {
```

Comment:
No need to mention module again in error message - we have structured logging now. Focus on mentioning the error message.
`events` can't be logged, it's a security risk - how did you overlooked it ? Is there specific reason to log events ?
cc: @sanpj2292

Reply:
> how did you overlooked it ? Is there specific reason to log events ?

The inclusion of `events` was a call we took to understand what is the event which is causing a problem. The reason is we don't have a lot of context on what kind of event was received to us & even though we get the error message, we would not be in a position to properly identify the problem and come up with a fix.

In this particular bugsnag issue, we saw that `sourceEvents` came in as a non-array which means there is some problem in adaption logic. And is it undefined, string, object -- we absolutely have no-clue

We will remove `events` from getting logged but we should definitely find an alternate (if not already in consideration or present) to address this

Reply:
@shrouti1507 let's remove `events` from the log getting emitted for an error